### ISSUE TITLE
Error when starting up.

### ISSUE THREAD

**Issue by boss69xd** [BullyWiiPlaza/JGeckoU#59]
![screenshot_1](https://user-images.githubusercontent.com/30028455/45933609-707ef680-bf5e-11e8-915b-4db1c2f5dca6.png)

I get this error every time i start up JGecko U.

**Comment by BullyWiiPlaza** [comment]
Yes, this happens to everyone because the URL has migrated to `https`. I will fix this soon.

**Comment by GeorgeGamingYT** [comment]
Woow u fixed it  -.-

![img_20180925_005247](https://user-images.githubusercontent.com/43551175/45985366-cd150b00-c05d-11e8-87fb-1ff4a51d209c.jpg)

**Comment by BullyWiiPlaza** [comment]
You need to to re-download it.

**Comment by BullyWiiPlaza** [comment]
[`This`](https://github.com/BullyWiiPlaza/JGeckoU/blob/master/JGecko%20U.jar?raw=true) is download link. I just checked it again. Nobody else still has this problem but you because you failed to download the newest version.

Thanks.

**Comment by GeorgeGamingYT** [comment]
Sorry Sir Last Comments
It works but I everytime I load code list nothing appears

**Comment by BullyWiiPlaza** [comment]
You have to copy the new `JGecko U.jar` file to the old folder or add new codes. Now please don't go any more off-topic. This issue isn't for asking user questions.
